<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListListDialog</class>
 <widget class="QDialog" name="ListListDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>List Manager</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QPushButton" name="NewListBtn">
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="EditBtn">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="DuplicateBtn">
     <property name="text">
      <string>Duplicate</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="FinishBtn">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="ListsLabelName">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Lists</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListView" name="listListView"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="ExportBtn">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="ImportListBtn">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="RemoveBtn">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listListView</tabstop>
  <tabstop>ImportListBtn</tabstop>
  <tabstop>ExportBtn</tabstop>
  <tabstop>RemoveBtn</tabstop>
  <tabstop>NewListBtn</tabstop>
  <tabstop>EditBtn</tabstop>
  <tabstop>DuplicateBtn</tabstop>
  <tabstop>FinishBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
